src/PrestaShopBundle/Resources/views/Admin/Product/ProductPage/Forms/form_combination.html.twig

```diff
@@ -35,7 +35,19 @@
         </div>
     </td>
     <td class="attribute-finalprice text-sm-right">
-      <span data-price="{{ form.vars.value.final_price }}" data-uniqid="{{ form.vars.value.id_product_attribute }}">{{ form.vars.value.final_price }}</span> {{ default_currency }}
+      <div>
+        <span class="final-price" data-price="{{ form.vars.value.final_price }}" data-uniqid="{{ form.vars.value.id_product_attribute }}">{{ form.vars.value.final_price }}</span> {{ default_currency }}
+      </div>
+      {% if 'PS_USE_ECOTAX'|configuration != 0 %}
```

Comment:
I did not know we could do that 😄

Reply:
Yep, me neither! That's quite useful actually!